frontend/src/components/common/Tabs.tsx

```diff
@@ -10,13 +10,13 @@ function a11yProps(index: number) {
   };
 }
 
-export interface Tab {
+export interface TabItem {
```

Comment:
This was renamed because Tab interface was shadowing the imported Tab.

Reply:
You could import it as `MuiTab`

Reply:
Ah yes, that's better.